package.json

```diff
@@ -15,16 +15,17 @@
   "author": "TripleA-Organisation",
   "license": "MIT",
```

Comment:
I missed this in #2.  Could you please change it to `AGPL-3.0-or-later`?

Reply:
Yeah, of course.
MIT will still continue to be my license of choice though ^^
I really don't care what my code is used for if anything